coderd/database/migrations/000277_workspace_next_start_at.up.sql

```diff
@@ -0,0 +1,38 @@
+ALTER TABLE ONLY workspaces ADD COLUMN IF NOT EXISTS next_start_at TIMESTAMPTZ DEFAULT NULL;
```

Comment:
Should there be an index on this colum so `GetWorkspacesEligibleForTransition` doesn't require a sequential scan?